pkg/gadgets/tcptracer/tracer/core/bpf/tcptracer.bpf.c

```diff
@@ -181,8 +200,11 @@ exit_tcp_connect(struct pt_regs *ctx, int ret, __u16 family)
 	if (!fill_tuple(&tuple, sk, family))
 		goto end;
 
+	task = (struct task_struct*)bpf_get_current_task();
```

Comment:
I think kernel style needs a space between the cast and the casted.

Reply:
I didn't. I tried and it reports many errors and warnings in other gadgets too, so I'll skip this time. I created https://github.com/kinvolk/inspektor-gadget/issues/718 to track this down.